services/pg-backup/docker-compose.yml.j2

```diff
@@ -20,6 +20,13 @@ services:
       placement:
         constraints:
           - node.labels.pgbackup==true
+      resources:
+        limits:
+          memory: 6G
```

Comment:
This looks scarry, maybe we need to make an exception here. 6GB just for a database backup wow :D 🗡️

Reply:
Changed to 512M